doc/integrator/upgrade_application.rst

```diff
@@ -28,14 +28,50 @@ Where ``<makefile>`` is your user make file (``<user>.mk``).
 
 Easy upgrading an application
 -----------------------------
+.. note:: For Windows:
 
-In the ``setup.py`` be sure that you don't specify a ``c2cgeoportal`` version
-that it don't allow to install the new ``c2cgeoportal`` egg.
+    If you are using Windows, you will have to execute some steps prior
+    to the instructions hereunder:
+
+    * Uninstall manually the old c2cgeoportal egg:
+
+        ``.build/venv/Scripts/pip uninstall c2cgeoportal-win``
```

Comment:
Because I am in a `.. note` section (see new line 31) and I could not make it work inserting `.. prompt:: bash` into it....